docs/ContribOperators.md

```diff
@@ -57,11 +58,14 @@ Do not modify directly.*
   * <a href="#com.microsoft.QLinearReduceMean">com.microsoft.QLinearReduceMean</a>
   * <a href="#com.microsoft.QLinearSigmoid">com.microsoft.QLinearSigmoid</a>
   * <a href="#com.microsoft.QLinearSoftmax">com.microsoft.QLinearSoftmax</a>
+  * <a href="#com.microsoft.QOrderedAttention">com.microsoft.QOrderedAttention</a>
```

Comment:
why add the attention op?

Reply:
These ops were added by this pr https://github.com/microsoft/onnxruntime/pull/12949, which didn't update the docs.